packages/protocol/test/forking/arbitrum/AaveV3.t.sol

```diff
@@ -16,14 +16,14 @@ contract AaveV3ArbitrumForkingTest is Routines, ForkingSetup {
   uint256 public constant BORROW_AMOUNT = 200 * 1e6;
 
   function setUp() public {
-    deploy(ARBITRUM_DOMAIN);
+    setUpFork(ARBITRUM_DOMAIN);
 
     aaveV3 = new AaveV3Arbitrum();
     ILendingProvider[] memory providers = new ILendingProvider[](1);
     providers[0] = aaveV3;
+    vm.makePersistent(address(aaveV3));
```

Comment:
Why do we have vm.makePersistent in this test and not the other ones?

Reply:
> Why do we have vm.makePersistent in this test and not the other ones? @DaigaroCota

@pedrovalido I forgot to remove it. Please do so in one of your provider implementations. It should not be affecting test passing.

Reply:
Got it will do. It doesnt affect, I was just curious 😅